sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/SdkInfo_AzureBotService.cs

```diff
@@ -1,20 +1,43 @@
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
 
-internal static partial class SdkInfo
+namespace Microsoft.Azure.Management.BotService
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
this is code auto generated by autorest@2.0.4413